source/includes/gcp/_k8_pods.md

```diff
@@ -212,15 +212,15 @@ Retrieve a list of all pods in a given [environment](#administration-environment
 
 Required | &nbsp;
 ------- | -----------
-`cluster_id` <br/>*string* | The id of the cluster in which to list the pods. 
+`cluster_id` <br/>*string* | The id of the cluster in which to list the pods.
 
 Attributes | &nbsp;
 ------- | -----------
-`id` <br/>*string* | The id of the pod.  
+`id` <br/>*string* | The id of the pod.
 `metadata` <br/>*object* | The metadata of the the pod
 `metadata.annotations` <br/>*map* | The annotations of the pod
 `metadata.creationTimestamp` <br/>*string* | The date of creation of the pod as a string
-`metadata.labels` <br/>*map* | The labels associated to the pod and there associated values
```

Comment:
It pains me that we're doing away with the punctuation but oh well :p This looks great Milt! I love how you solved both problems with one delete, ahha.